src/CONTRIBUTORS.js

```diff
@@ -3,10 +3,42 @@
  * Feel free to add yourself if you're not yet in the list.
  *
  * Using `require` for avatars so we don't have to keep a seperate list of imports disconnected from the maintainer definition.
- *
- * @type {{nickname: string, github: string, avatar: *}}
+ * 
+    EXAMPLE
+
+    export const NICKNAME = {
+      nickname: 'NICKNAME',
+      github: 'GITHUB_NAME',
+      avatar: require('./Images/IMAGE'),
+      desc: 'DESC',
+      maintainer: [
```

Comment:
This creates some duplication as maintainers are already defined in the specs' CONFIG file. Would it be possible to get the info from those instead? Alternatively we could define it here instead of in the CONFIG files, I'm not sure which would be prefered.

Reply:
I'll change that aswell.
Using the AVAILABLE_CONFIGS for importing the changelogs will allow me easily to access the specs maintainer. Would also mean we avoid going through all specs + files to make the maintainer contributor-based.